staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -876,9 +876,9 @@ type PodSecurityPolicy struct {
 	Spec PodSecurityPolicySpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
 }
 
-// Pod Security Policy Spec defines the policy enforced.
+// PodSecurityPolicySpec defines the policy enforced.
 type PodSecurityPolicySpec struct {
-	// privileged determines if a pod can request to be run as privileged.
+	// Privileged determines if a pod can request to be run as privileged.
```

Comment:
leave these lowercase... we want api documentation to match the json fields

Reply:
Thanks! I reverted these changes and modified other places to use lowercase. PTAL.